.env.sample

```diff
@@ -2,5 +2,8 @@
 ARK_GO_DIR=./ark-go
 ARKD_DIR=${ARK_GO_DIR}/ark/server
 ARKD_WALLET_DIR=${ARK_GO_DIR}/pkg/ark-wallet
```

Comment:
we can squash in this commit also the following one https://github.com/arkade-os/rust-sdk/pull/86/commits/fd3bea261ef2e945c2b2006bc0e3c0c0744e5ce8

Reply:
Good call. I've just cherry-picked it from your branch.